tools/build_config/build.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -o errexit  # exit early on any error
+set -o nounset  # raise error using unused variables
+
+FALSE=0
+TRUE=1
+CMAKE_GENERATOR="Unix Makefiles"
+HORACE_ROOT=$(realpath $(dirname "$0")/../..)
```

Comment:
it may be helpful to add some comments explaining these two magic path calculations